### ISSUE TITLE
Cannot see inspector

### ISSUE THREAD

**Issue by jay20162016** [godotengine/godot#58804]
### Godot version

Godot Engine v4.0.alpha3.official.256069eaf

### System information

mac

### Issue description

<img width="887" alt="Screen Shot 2022-03-05 at 3 33 29 PM" src="https://user-images.githubusercontent.com/20290260/156899167-edc1325a-d81e-4beb-8d88-03416e61e673.png">
Inspector is blocked by options of scene view

### Steps to reproduce

create label, click on it

**Comment by jay20162016** [comment]
Note toggling visibility to off will remove the components right of the trash can icon, making the inspector visible.

**Comment by Calinou** [comment]
This issue has the same cause as https://github.com/godotengine/godot/issues/26835. It doesn't occur in `3.x` since the Label font overrides feature isn't present there.

**Comment by KoBeWi** [comment]
Duplicate of #58052
There's a PR open that solves this case.
